appveyor.yml

```diff
@@ -28,7 +28,7 @@ build_script:
 
 test_script:
   - cmd: cd C:\projects\Mezz_Filesystem\build
-  - cmd: Filesystem_Tester xml
+  - cmd: Filesystem_Tester debugtests xml
```

Comment:
Can this be made to work with "thisprocess" instead? It should do the same thing but is more clear about intent.